spec/system/schools/school_targets_spec.rb

```diff
@@ -201,6 +204,49 @@
 
         it { expect(page).not_to have_link("View results") }
 
+        context 'and there are progress reports for both targets' do
+          let!(:expired_electricity_progress) { build(:fuel_progress, fuel_type: :electricity, progress: 0.99, target: 20, usage: 15) }
+          let!(:expired_gas_progress)         { build(:fuel_progress, fuel_type: :gas, progress: 0.59, target: 19, usage: 17) }
+          let!(:expired_last_generated)       { Date.yesterday }
```

Comment:
could be within the :expired_target let? should it be the same as target_date?

Reply:
I kept them separate as I think its easier to check the values that way, then have multiple objects built in one larger let block.